### ISSUE TITLE
Unexpected token 'U', "Unexpected"... is not valid JSON error

### ISSUE THREAD

**Issue by gksllc** [stackblitz/bolt.new#1534]
### Describe the bug

will not load code/page

Unexpected token 'U', "Unexpected"... is not valid JSON

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-tunxbx

### Steps to reproduce

starting up the conversation won't load the code/page

### Expected behavior

should load page

### Screen Recording / Screenshot

error: Unexpected token 'U', "Unexpected"... is not valid JSON

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 90019762,
  "usedJSHeapSize": 72969510,
  "jsHeapSizeLimit": 4294705152
}
Username = Gksllc
Chat ID = 51d2a3eb5f00
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by Alm0stEthical** [comment]
+1 Same issue when importing from StackBlitz

**Comment by gksllc** [comment]
i can load my files now. but it didn't initially load them until i told bolt to load the files. also, it seems to have deployed to splendid-croissant when i never deployed it

**Comment by kc0tlh** [comment]
@gksllc Thanks for the report and apologies for the frustrating experience! The fix is explained in #1471
